lib/src/remote_frame_buffer_isolate_messages.dart

```diff
@@ -15,6 +15,7 @@ class RemoteFrameBufferIsolateInitMessage
     required final String hostName,
     required final Option<String> password,
     required final int port,
+    required final Duration? timeout,
```

Comment:
suggestion: I want to avoid all null-aware code in this package. My suggestion would be to make this an `Option<Duration>` instead.